graph/pkg/service/v0/driveitems.go

```diff
@@ -92,6 +92,30 @@ func (g Graph) getDriveItem(ctx context.Context, root *storageprovider.ResourceI
 	return cs3ResourceToDriveItem(res.Info)
 }
 
+func (g Graph) getRemoteItem(ctx context.Context, root *storageprovider.ResourceId, baseURL *url.URL) (*libregraph.RemoteItem, error) {
+	client := g.GetGatewayClient()
+
+	ref := &storageprovider.Reference{
+		ResourceId: root,
+	}
+	res, err := client.Stat(ctx, &storageprovider.StatRequest{Ref: ref})
+	if err != nil {
+		return nil, err
+	}
+	if res.Status.Code != cs3rpc.Code_CODE_OK {
+		// Only log this, there could be mountpoints which have no grant
+		g.logger.Debug().Msg(res.Status.Message)
+		return nil, nil
```

Comment:
Are we sure we want to `return nil, nil` here? One will be forced to use this func like:
```golang
item, err := getRemoteItem(...)
if err != nil {
    // handle error case
}
if item == nil {
    // handle hidden unsuccessful stat
}
```
Note that the function caller needs to know what happens under the hood in `getRemoteItem` func

Reply:
ok, let us move the reponsability to hadle the error to the caller.